lib/wor/paginate/formatter.rb

```diff
@@ -32,8 +32,13 @@ def paginated_content
       end
 
       def serialized_content
-        return paginated_content.as_json if serializer.blank?
-        paginated_content.map { |item| serializer.new(item) }
+        return paginated_content.map { |item| serializer.new(item) } if serializer.present?
```

Comment:
I think this is not responsibility of the formatter. The formatter should only build the json response. What if we create our serializer that checks this condition and the formatter only talks with its interface without knowing that it has to check if the user has sent a serializer or not?